app/models/user.rb

```diff
@@ -1,11 +1,49 @@
 class User < ActiveRecord::Base
   concerned_with :validation, :findability, :settings, :statistics
 
+  validates :email,
+    format: {
+      with: /@/,
```

Comment:
I was sort of curious why a better regexp wasn't recommended in the guide like `URI::MailTo::EMAIL_REGEXP` https://stackoverflow.com/a/49925333

Reply:
Ahh I see, they give what was used in Authlogic 4. My vote would be to move to that or move to something standard like the EMAIL_REGEXP.